test/check-application

```diff
@@ -474,21 +472,54 @@ class TestApplication(testlib.MachineCase):
         self.assertIn('rw', rwmnt)
         self.assertIn(rwdir[4:], rwmnt)
 
+        if m.image != "fedora-29":
+            b.wait(lambda: "Hello World" in m.execute("podman logs busybox-with-tty"))
```

Comment:
This previously worked on Fedora 29, too. Why would it stop working now?

Reply:
right, it wouldn't I just moved it and later on added condition. I'll move it back